### ISSUE TITLE
Migrations in the old way

### ISSUE THREAD

**Issue by atefrihane** [Artem-Schander/L5Modular#58]
Hey guys

How can I generate migrations in the standard way like in the previous versions ?
I've played around with config files but no way

**UPDATE :**

I had to edit this method here in **MigrateMakeCommand** Class

```
 protected function getMigrationPath()
    {
       
        $migrationPath = $this->laravel['path'] .'/' . $this->getConfiguredFolder('migrations');
   
        if (! is_null($targetPath = $this->input->getOption('path'))) {
            return $migrationPath . '/' . $targetPath;
        }

        return $migrationPath;
    }
```

is there a cleaner way to make this change ?

**Comment by Artem-Schander** [comment]
Hi @atefrihane,

If I understand correctly, you could use Laravels make method.
Like: `php artisan make:migration --create=some_table create_some_table`

I hope this helps. Otherwise feel free to reopen this issue.

**Comment by atefrihane** [comment]
@Artem-Schander  Yes I know I can do that manually with the make migration command..
but what I meant is to generate it automatically in app/database folder like ( in <= 1.4 version)..
Thank you for your answer anyway

**Comment by Artem-Schander** [comment]
What do you mean by "automatically"?
In both scenarios you have to execute an artisan command, either `make:migration` or `make:module:migration`.
Or am I missing something here?

**Comment by atefrihane** [comment]
@Artem-Schander 
what I meant is when I execute`make:module ModuleName` in the old versions it creates a   ModuleName folder + migration file  in `app/database` ..
For the current version when you set `migration=>true` in  config file it , the migration gets created  inside the module folder itself..
That's why I was asking if I can do it in the old way..
I hope it's clear!

**Comment by Artem-Schander** [comment]
Ah, now I got it.
Not my brightest moment ^^
I'll see what I can do.
